FastSimulation/ShowerDevelopment/interface/FastHFShowerLibrary.h

```diff
@@ -45,61 +46,32 @@ class FastHFShowerLibrary {
 
 public:
 
-  struct Hit {
-    Hit() {}
-    G4ThreeVector             position;
-    int                       depth;
-    double                    time;
-  };
-
   void                initRun();
   void       const    initHFShowerLibrary(const edm::EventSetup& );
   void                recoHFShowerLibrary(const FSimTrack &myTrack);
-  std::vector<Hit>    getHits(const G4ThreeVector & p, const G4ThreeVector & v,
-                              int parCode, double parEnergy, bool &ok, 
-                              double weight, bool onlyLong=false);
-  const std::map<CaloHitID,float>& getHitsMap() { return hitMap; };
-  bool                isItinFidVolume (G4ThreeVector&);
 
-protected:
+  std::vector<HFShowerLibrary::Hit> getHits(G4ThreeVector & p, G4ThreeVector & v,
```

Comment:
@Vlandr57 

when you move the parCode condition into HFShowerLibrary::fillHits,
the parcode condition can be removed from FastHFShowerLibrary::getHits.

The 2 remaining lines of code inside this function better move to FastHFShowerLibrary::recoHFShowerLibrary